Assets/MixedRealityToolkit/_Core/Definitions/Utilities/MixedRealityPose.cs

```diff
@@ -25,7 +25,7 @@ public SixDof(Vector3 position, Quaternion rotation)
         /// Constructor.
         /// </summary>
         /// <param name="position"></param>
-        public SixDof(Vector3 position)
+        public MixedRealityPose(Vector3 position)
```

Comment:
Recommend 
public MixedRealityPose(Vector3 position) : this(position, Quaternion.identity) { }